pulp/apis/coin_api.py

```diff
@@ -416,6 +417,7 @@ def __init__(
                 logPath=logPath,
                 mip_start=mip_start,
                 timeMode=timeMode,
+                maxNodes=None,
```

Comment:
here you need to put `maxNodes=maxNodes`

Reply:
> here you need to put `maxNodes=maxNodes`

Thanks for reviewing, made the change ! @pchtsp